srcDLL/DllMain.cpp

```diff
@@ -124,13 +126,9 @@ void OnShutdown()
 /**
 Prepares all vol files found within the supplied relative directory from the Outpost 2 executable
 for inclusion in Outpost 2. Does not recursively search subdirectories.
-
-@param relativeDirectory A directory relative to the Outpost 2 exectuable. Default value is an empty string.
 */
-void LocateVolFiles(const std::string& relativeDirectory)
+void LocateVolFiles(const std::string& absoluteDirectory)
 {
-	const auto absoluteDirectory = (fs::path(GetExeDirectory()) / relativeDirectory).string();
```

Comment:
I was thinking perhaps a `basePath` argument could be added. Paths could be given relative to the `basePath`. We could offer a one parameter overload that uses `GetExeDirectory()` for the `basePath`.

Reply:
What about:

```C++
AddVolFilesFromAbsoluteDirectory(const std::string& absoluteDirectory);

// relativeDirectory is from the directory containing the Outpost 2 exectuable
AddVolFilesFromRelativeDirectory(const std::string& relativeDirectory);
```

Reply:
Hmm, it seems my original idea here was a bit fuzzy.

I was thinking for a generalized function:
```cpp
ResultType FindFilesInFolder(const std::string& relativeDirectory, const std::string& basePath) {
  // Main implementation here
}

ResultType FindFilesInFolder(const std::string& relativeDirectory) {
  return FindFilesInFolder(relativeDirectory, GetExeDirectory());
}
```

The resulting paths would be relative to the `basePath` parameter.

The above ignores the details of how the paths are returned. Maybe through a `std::vector`, or maybe to a callback, which might be passed as an extra parameter. It also ignores other parameters that might be added for a generalized function, such as a filename extension to look for.

----

Whether we can form of VOL list from relative or absolute folders is a concern I had not considered. I don't even really remember why the system was designed the way it was to only take relative paths.

Reply:
I believe Outpost 2 uses relative paths to the vol files internally when loading.